### DISCUSSION TITLE
Pusher broadcast ApiErrorException empty

### DISCUSSION THREAD

**Discussion by biscolab** [laravel/framework#45866] [Q&A]
Hi! I'm experiencing a strange issue using Pusher.
`$this->pusher->trigger` throws and `ApiErrorException` with an empty `$e->getMessage()`

Trace:
Illuminate\Broadcasting\BroadcastException
Pusher error: .
(Illuminate\Broadcasting\Broadcasters\PusherBroadcaster::broadcast at line 128)

This error is connected to a scheduled artisan command running several times per day with the same data...and often fails as shown...Any clue?

I also checked the Pusher error log...still empty...

Thanks

**Comment by henzeb** [comment]
I think you should look at Pusher. instead. Can you run the call in postman or similar tool and see what happens?

**Reply by biscolab** [reply]
Hi @henzeb, thanks for your help..."unfortunately" every test API call has been finished successfully
